doc/source/development.rst

```diff
@@ -63,8 +63,8 @@ Post-release
 Roadmap
 =======
 
-* From Molecule v2.20 `GitHub Milestones`_ are used to track each release
+* From Molecule v2.20 the `GitHub Project`_ tool is used to track each release.
 
 .. _`PyPI`: https://pypi.org/project/molecule
-.. _`GitHub Milestones`: https://github.com/ansible/molecule/milestones
+.. _`GitHub Project`: https://github.com/ansible/molecule/projects
```

Comment:
Can also put it into this section https://github.com/ansible/molecule/blob/master/setup.cfg#L15 so that it'd be on PyPI as well.

Reply:
Yep! Submitted https://github.com/ansible/molecule/pull/1792.